tests/Doctrine/Tests/Common/Annotations/AbstractReaderTest.php

```diff
@@ -387,8 +387,11 @@ public function testInvalidAnnotationUsageButIgnoredClass()
         $reader = $this->getReader();
         $ref = new \ReflectionClass('Doctrine\Tests\Common\Annotations\Fixtures\InvalidAnnotationUsageButIgnoredClass');
         $annots = $reader->getClassAnnotations($ref);
-
-        $this->assertEquals(2, count($annots));
+        if ($annots[0] instanceof IgnoreAnnotation) {
+            $this->assertEquals(2, count($annots));
```

Comment:
I'm in favour of `assertCount()`, but when creating pull requests it's considered good behaviour to conform to the style of the project. 

Please create a pull request changing all the assertEquals into the better assertCount and when that gets merged first i'll gladly change the assertions.

PS. In my personal project I use `assertSame` and don't use `assertEquals` at all, as it doesn't catch subtle bugs like `$this->assertEquals("1", true);`